packages/components/stories/Map.stories.ts

```diff
@@ -43,6 +43,13 @@ type Story = StoryObj<MapProps>;
 export const GeoJSONPolygons: Story = {
   name: 'GeoJSON polygons map',
   args: {
+    tile : {
```

Comment:
We should document this new parameter on the story

Reply:
will do it next @demenech